package.json

```diff
@@ -17,8 +17,8 @@
         "knex": "yarn tsn node_modules/.bin/knex --knexfile knexfile.ts",
         "migrate": "yarn knex migrate:latest && yarn typeorm migration:run",
         "lint": "eslint .",
-        "prettify": "yarn prettier --write \"**/*.{tsx,ts,jsx,js,json,md,html,css,scss}\"",
-        "prettify:check": "yarn prettier --check \"**/*.{tsx,ts,jsx,js,json,md,html,css,scss}\"",
+        "prettify": "yarn pretty-quick --pattern \"**/*.{tsx,ts,jsx,js,json,md,html,css,scss}\"",
+        "prettify:check": "yarn pretty-quick --check --pattern \"**/*.{tsx,ts,jsx,js,json,md,html,css,scss}\"",
```

Comment:
Hm, does this also work in CI or will it just not detect any changes?

Reply:
Ah, good question. I think it should work since it should still compare to master, but I'm assuming a lot there so let me test it.

Reply:
I needed to change Travis to also checkout `master` so it has something to compare against. That ends up being pretty fast as usually there is mostly overlap between the 2 branches. Travis only pulls to a depth of 50, max, so I think in theory if a branch has diverged by more than that the test will pass (since it won't have a common root to compare to), but I think if someone's branch has diverged by more than 50 commits prettier might not be the biggest problem. 👍 

Anyway, my Travis test branch now failed as expected, so hopefully it should work in CI going forward.